extension/js/common/composer.ts

```diff
@@ -1181,13 +1181,13 @@ export class Composer {
   }
 
   private renderReplyMsgComposeTable = async (method: 'forward' | 'reply' = 'reply'): Promise<void> => {
-    this.urlParams.subject = `${(method === 'reply' ? 'Re' : 'Fwd')}: ${this.urlParams.subject}`;
     this.S.cached('prompt').css({ display: 'none' });
     this.S.cached('input_to').val(this.urlParams.to.join(',') + (this.urlParams.to.length ? ',' : '')); // the comma causes the last email to be get evaluated
     await this.renderComposeTable();
     if (this.canReadEmails) {
       const determined = await this.app.emailProviderDetermineReplyMsgHeaderVariables();
       if (determined) {
+        this.urlParams.subject = `${(method === 'reply' ? 'Re' : 'Fwd')}: ${this.urlParams.subject}`;
```

Comment:
This is a nuanced thing, and I'm not super sure what is the right way to do it, but I think maybe putting it above in the original position may be a bit better.

I'm imagining a situation when user is intending to send a reply, but for whatever reason, the right headers cannot be found. In such cases, I would still want subject to show `Re: ` so that the intention is visible that the user meant to reply.

If this ever causes normal replies to be missing headers, we should catch it in the ci tests.

What do you think?

Reply:
I'm not sure, 

How can they reply to unexisting `threadId`?
Is there a way to reproduce it using default Gmail Reply box and check how they solved this.

Reply:
1) Find a conversation to reply to
2) Open a reply box
3) Go to your phone and delete the convo, then delete it from trash
4) wait a bit for the phone to sync changes to Gmail server
5) (back on desktop) send

I don't worry too much about what Gmail does in this scenario, it's too infrequent and not really central to the UX experience etc.

Reply:
Also, we could simply sometimes parse out the thread id wrong. That could happen.

Reply:
Got it,

If we can minimize chances of wrong parsing `threadId` then I think it's the right way if we remove `Re: ` prefix.
That's what I would expect from Flowcrypt or Gmail. (I mean I know that something wrong with the thread I'm replying to).

Reply:
Ok then. Fine with me.